hack/run-kiali.sh

```diff
@@ -680,7 +704,12 @@ start_port_forward_component() {
     else
       startmsg "The port-forward to ${COMPONENT_NAME} is being started on [${EXPECTED_URL}]"
       set -m
-      (while true; do ${CLIENT_EXE} port-forward -n ${ISTIO_NAMESPACE} ${PORT_FORWARD_DEPLOYMENT} --address=127.0.0.1 ${LOCAL_REMOTE_PORTS} > /dev/null; warnmsg "${COMPONENT_NAME} port-forward died - restarting on [${EXPECTED_URL}]"; sleep 1; done) &
+      if [ ${COMPONENT_NAME} == "Tracing" ]; then
```

Comment:
You should always wrap env vars in quotes  when inside if stmts to ensure you always get an explicit argument
```suggestion
      if [ "${COMPONENT_NAME}" == "Tracing" ]; then
```